src/extension.ts

```diff
@@ -1075,32 +1067,21 @@ async function refreshExplorer() {
     await vscode.commands.executeCommand("extension.vsKubernetesRefreshExplorer");
 }
 
-async function reportDeleteResult(resourceId: string, shellResult: ShellResult) {
-    if (shellResult.code !== 0) {
-        await vscode.window.showErrorMessage(`Failed to delete resource '${resourceId}': ${shellResult.stderr}`);
-        return;
-    }
-    await vscode.window.showInformationMessage(shellResult.stdout);
-    refreshExplorer();
-}
-
 const deleteKubernetes = async (explorerNode? : explorer.ResourceNode) => {
     if (explorerNode) {
         const answer = await vscode.window.showWarningMessage(`Do you want to delete the resource '${explorerNode.resourceId}'?`, ...deleteMessageItems);
         if (answer.isCloseAffordance) {
             return;
         }
-        const shellResult = await kubectl.invokeAsyncWithProgress(`delete ${explorerNode.resourceId}`, `Deleting ${explorerNode.resourceId}...`);
-        await reportDeleteResult(explorerNode.resourceId, shellResult);
+        kubectl.invokeInTerminal(`delete ${explorerNode.resourceId}`);
```

Comment:
Hmm, this is a case where I don't much like the terminal experience.  It requires the user to pay attention to a noisy text window to see the outcome, whereas the old way (info message for success and error message for failure) gave much more obvious feedback.  (Scale would probably be another example of this.)  It's fine to point the user to the terminal window output if they want to grovel through the error details, but it would be good to surface the success/failure indicators more simply.  `kubectl` is an implementation detail, which it's useful to surface but shouldn't be in the user's face all the time.

Reply:
yeah, i saw the ux agreement in yesterday's meeting, have reverted to the old design.